core/src/Pos/Core/Update/Proposal.hs

```diff
@@ -151,7 +160,25 @@ checkUpdateProposal pm it = do
     unless (checkSig pm SignUSProposal (upFrom it) toSign (upSignature it))
         (throwError "UpdateProposal: invalid signature")
 
+----------------------------------------------------------------------------
+-- Software Update Vote
+--
+-- N.B. VoteId needs to be defined in here or else its Buildable instance
+-- will be an orphan.
+----------------------------------------------------------------------------
+
+-- | ID of a voter and its decision regarding a specific software update
+-- proposal
+type VoteId = (UpId, PublicKey, Bool)
```

Comment:
The type definition for `VoteId` was moved here (`Pos.Core.Update.Proposal`) due to the fact that its [`Buildable` instance](#diff-ce08f977beb84eec59cfb3277ab8c11cR174) would be an orphan if not defined in the same module as `UpId` (`VoteId` is a type alias for `(UpId, PublicKey, Bool)`)

Reply:
Seems reasonable.